core/test-runtime/client/src/lib.rs

```diff
@@ -97,12 +96,33 @@ pub type LightExecutor = client::light::call_executor::RemoteOrLocalCallExecutor
 pub struct GenesisParameters {
 	support_changes_trie: bool,
 	heap_pages_override: Option<u64>,
+	extra_storage: Vec<(Vec<u8>, Vec<u8>)>,
+}
+
+impl GenesisParameters {
+	fn genesis_config(&self) -> GenesisConfig {
+		GenesisConfig::new(
+			self.support_changes_trie,
+			vec![
+				sr25519::Public::from(Sr25519Keyring::Alice).into(),
+				sr25519::Public::from(Sr25519Keyring::Bob).into(),
+				sr25519::Public::from(Sr25519Keyring::Charlie).into(),
+			], vec![
```

Comment:
```suggestion
			],
			vec![
```

Reply:
Hah, had this change in some other stashed PR 👍